src/core/utils/misc.cpp

```diff
@@ -260,7 +271,7 @@ int Utils::Misc::pythonVersion()
         if (output.isEmpty())
             output = python_proc.readAllStandardError();
         const QByteArray version_str = output.split(' ').last();
-        qDebug() << "Python version is:" << version_str.trimmed();
+        Logger::instance()->addMessage(QCoreApplication::translate("misc", "Python version: %1").arg(QString(version_str)), Log::INFO);
```

Comment:
I'm putting the comment here but not related with this line:
If python is not detected (not installed) we don't display anything in the log, maybe we should.